ui/src/actions/__tests__/actions-ui.test.js

```diff
@@ -1,15 +1,18 @@
 import * as actions from '../actions-ui';
-import { REVERSE_SORT } from '../types';
+import { APPS_SORT } from '../types';
```

Comment:
I wonder is this descriptive enough to distinguish it from the sort we will be doing on the the AppDetails table, or do you have a suggestion on what the sort action for AppDetails should be named so we keep them consistent?

Reply:
I would suggest APP_SORT for the app table